cms-xcache/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM opensciencegrid/xcache:fresh
+
+RUN yum -y install xrootd --enablerepo=osg-minefield
```

Comment:
This line should be unnecessary if we're using the xcache container as a base

Reply:
Deleted.